iati_dashboard/settings.py

```diff
@@ -86,8 +86,20 @@
     "django_filters",
     "django.contrib.contenttypes",
     "django.contrib.auth",
+    "drf_spectacular",
 ]
 
+REST_FRAMEWORK = {
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+}
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "Your Project API",
```

Comment:
Is the plan to change this later, or does it pull override these settings somewhere? It's also got the same placeholder TITLE and DESCRIPTION in the `schema.yml`.

Reply:
Yep!